redfish-core/lib/log_services.hpp

```diff
@@ -1596,10 +1596,16 @@ inline void updateProperty(const crow::Request& req,
                            const std::string& entryId)
 {
     std::optional<bool> resolved;
-    if (!json_util::readJsonPatch(req, asyncResp->res, "Resolved", resolved))
+    std::optional<nlohmann::json> oemObject;
+#ifdef BMCWEB_ENABLE_IBM_MANAGEMENT_CONSOLE
+    std::optional<bool> managementSystemAck;
+#endif
+    if (!json_util::readJsonPatch(req, asyncResp->res, "Resolved", resolved,
+                                  "Oem", oemObject))
```

Comment:
Probably better if 

 "Oem", oemObject
 
 was also behind the flag, oh well

Reply:
Yea, you're right.  Though I guess for us anyway that flag will always be set.